x/dex/keeper/keeper.go

```diff
@@ -47,3 +48,77 @@ func (k Keeper) Logger(ctx sdk.Context) log.Logger {
 func (k Keeper) GetAuthority() string {
 	return k.authority
 }
+
```

Comment:
Since we are now just using events and there is not an issue with overloading cardinality it would be nice to add an additional field for pairId, so that have the option to segregate all of these events by specific pairs.